lib/omniauth/strategies/saml.rb

```diff
@@ -27,6 +27,7 @@ def self.inherited(subclass)
         first_name: ["first_name", "firstname", "firstName"],
         last_name: ["last_name", "lastname", "lastName"]
       }
+      option :relay_state
```

Comment:
Options that are not for `ruby-saml` settings are also passed along to the `Settings` object, but the initializer seems to ignore any that don't have a setter. Could filter the options using `OneLogin::RubySaml::Settings.instance_methods.select { |name| name.to_s.end_with?("=") }` but this seems kindof redundant.